linux/LinuxProcess.c

```diff
@@ -369,6 +757,18 @@ long LinuxProcess_compare(const void* v1, const void* v2) {
       return ((long)p2->ctxt_diff - (long)p1->ctxt_diff);
    case SECATTR:
       return strcmp(p1->secattr ? p1->secattr : "", p2->secattr ? p2->secattr : "");
+   case COMM:
+      return strcmp(LinuxProcess_getCommandStr((const Process *)p1), LinuxProcess_getCommandStr((const Process *)p2));
```

Comment:
Candidate for `Process_getCommand(p)`?

Reply:
Agred: In fact let COMM default to Process_compare(), which now uses Process_getCommand().